modules/importing-repositories/procedure-data-handoff-and-custom-workflow-design.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="full-data-handoff-and-custom-workflow-design"]
+
+= Data handoff and custom workflow design
+
+When you configure the Bulk Import plugin by setting the importAPI field to 'scaffolder', the Bulk Import Backend ensures all necessary context is passed directly to the Orchestrator.
+
+Your control over input data:: The Orchestrator workflow receives the input data required to automate processing for every selected repository. The payload includes, at minimum, the complete list of repository URLs and the user identity.
++
+[source,yaml]
+----
+{
+  "repositories": [
+    "[https://github.com/my-org/repo-a](https://github.com/my-org/repo-a)",
+    "[https://github.com/my-org/repo-b](https://github.com/my-org/repo-b)",
+    "[https://github.com/my-org/repo-c](https://github.com/my-org/repo-c)"
```

Comment:
Consired adding a description list after the code block to explain the list format.

It seems to be a list in JSON format, of repository URLS in Markdown format? Please clarify with the SMEs.